README.md

```diff
@@ -1,35 +1,806 @@
-Active Choices Plug-in
-=================
+Older versions of this plugin may not be safe to use. Please review the
+following warnings before using an older version:
 
-A Jenkins **UI plugin for selecting one or multiple options for a job parameter**. It provides a number of capabilities
-in a single plugin some, but not all, of which can be found amongst several other plugins. This project was previously
-called Uno Choice Plug-in, while under the BioUno project.
+-   [Arbitrary code execution
+    vulnerability](https://jenkins.io/security/advisory/2017-04-10/){.external-link}
```

Comment:
I think I mentioned yesterday about an issue I had in image-gallery-plugin with CSS classes not being removed by the Wiki conversion tool.

If you open this markdown file in github (link for this PR's file: https://github.com/jenkinsci/active-choices-plugin/blob/ce86251b6ca2c14d3afdae940db541c7afbe1631/README.md), you should be able to see the CSS classes still there.

They appear next to pretty much every image too. So need to find-replace by nothing.